wandb/sdk/launch/deploys/Dockerfile

```diff
@@ -7,7 +7,7 @@ RUN apt-get update && apt-get upgrade -y \
     && apt-get clean && rm -r /var/lib/apt/lists/*
 
 # required pip packages
-RUN pip install --no-cache-dir git+https://github.com/wandb/client.git@gong/deployable-launch-agent#egg=wandb[launch] kubernetes
+RUN pip install --no-cache-dir git+https://github.com/wandb/client.git@gong/deployable-launch-agent#egg=wandb[launch] kubernetes google-cloud-storage boto3
```

Comment:
Shouldn't these libraries come as a part of the `wandb[launch]` extras?

Reply:
Yes, I'll add them there